Refactor to pyansys standards (#140)
pyansys/pydpf-post: docs/source/user_guide/extending_to_core.rst

suited for representing more general data transformations with the use
of powerful, scalable operators.

DPF-Core is equipped to access data from Ansys solver result files as well as
third-party formats. Various `operators
<https://dpfdocs.pyansys.com/operator_reference.html>`_ are available to

> facilitate data manipulation and transformation. DPF-Core allows arbitrarily
> complex workflows to be assembled from simpler building blocks, by chaining

Reviewer: ```suggestion
facilitate data manipulation and transformation. DPF-Core allows you to arbitrarily
assemble complex workflows from simpler building blocks, by chaining
```

Export VTK
----------

> The following code shows how to **export a fields container in VTK format**:

Reviewer: ```suggestion
The following code shows how to export a fields container in VTK format:
```

Reviewer: There is no need to use bold here.

.. code:: python

>     Instantiate the solution object.

Reviewer 2: ```suggestion
    # Instantiate the solution object.
```

    >>> from ansys.dpf import post
    >>> from ansys.dpf.post import examples
    >>> solution = post.load_solution(examples.multishells_rst)

>     Instantiate a result object. This is a displacement result.

Reviewer 2: ```suggestion
    # Instantiate a result object. This is a displacement result.
```

    >>> displacement = solution.displacement()

>     This is the result data (data container).

Reviewer 2: ```suggestion
    # This is the result data (data container).
```

    >>> norm = displacement.norm

>     # The result fields_container is extracted from the result data.

Author: ```suggestion
    The result fields_container is extracted from the result data.
```

    >>> fields_container = norm.result_fields_container
    

>     # Now the Core API need to be imported.

Author: ```suggestion
    Now the Core API needs to be imported.
```

    >>> from ansys.dpf import core

    # A dedicated operator needs to be instantiated.

    >>> vtk_operator = core.Operator("vtk_export")

    # Connections must be set.

    >>> vtk_operator.inputs.mesh.connect(solution.mesh)
    >>> vtk_operator.inputs.file_path.connect("vtk_example.vtk")
    >>> vtk_operator.inputs.fields1.connect(fields_container)

>     # Run the operator.

Author: ```suggestion
    Run the operator.
```

.. code:: python

>     # Instantiate the solution object

Author: ```suggestion
    Instantiate the solution object
```

    >>> from ansys.dpf import post
    >>> from ansys.dpf.post import examples
    >>> solution = post.load_solution(examples.multishells_rst)

    # Instantiate a result object. This is a displacement result.

    >>> displacement = solution.displacement()

>     # This is the result data (data container)

Author: ```suggestion
    This is the result data (data container)
```

    >>> norm = displacement.norm

>     # Extract the result fields_container from the result data.

Author: ```suggestion
    Extract the result fields_container from the result data.
```

    >>> fields_container = norm.result_fields_container
    
    # Now the Core API needs to be imported.

    >>> from ansys.dpf import core

>     # Initialize a dedicated operator.

Author: ```suggestion
    Initialize a dedicated operator.
```

    >>> h5_operator = core.Operator("serialize_to_hdf5")

>     # Set the connection.

Author: ```suggestion
    Set the connection.
```

    >>> h5_operator.inputs.mesh.connect(solution.mesh)
    >>> h5_operator.inputs.file_path.connect("hdf5_example.h5")
    >>> h5_operator.inputs.data.connect(fields_container)

>     # Evaluate the the operator.

Author: ```suggestion
    Evaluate the the operator.
```
